sympy/solvers/solveset.py

```diff
@@ -281,26 +281,6 @@ def _domain_check(f, symbol, p):
                     for g in f.args])
 
 
-def _is_finite_with_finite_vars(f, domain=S.Complexes):
```

Comment:
I don't think it is useful. Because if `domain = S.complexes` then it returns true for some cases and not look correct.